pkgs/applications/audio/aacgain/default.nix

```diff
@@ -1,27 +1,26 @@
 { lib
 , stdenv
-, fetchFromGitHub
+, fetchgit
 , cmake
 , autoconf
 , automake
 , libtool
-, faad2
 , mp4v2
 }:
 
 stdenv.mkDerivation rec {
   pname = "aacgain";
   version = "2.0.0";
 
-  src = fetchFromGitHub {
-    owner = "dgilman";
-    repo = pname;
+  src = fetchgit {
```

Comment:
`fetchFromGitHub` with `fetchSubmodules = true` should normally work. Did you remove the old hash before building the package?

Reply:
I didn't, thanks for the hint! I assumed the invocation with `fetchSubmodules = true;` would have to result in an entirely new derivation anyway (as `fetchFromGitHub` should then switch to `fetchgit` internally), but that seems to indeed have been the issue. I guess there's some optimization that relies entirely on the specified hash? TIL.

I force pushed a smaller change accordingly, c.f. a2ce9baea0797604fd6e0a5fa23540aacb76306f